cdap-app-templates/cdap-etl/hydrator-spark-core/src/main/java/io/cdap/cdap/etl/spark/batch/RDDCollection.java

```diff
@@ -123,6 +126,54 @@ public RDDCollection(JavaSparkExecutionContext sec, JavaSparkContext jsc, SQLCon
       }
       seenRequired = seenRequired || toJoin.isRequired();
 
+      // UUID for salt column name to avoid name collisions
+      String saltColumn = UUID.randomUUID().toString();
+      if (joinRequest.isDistributionEnabled() && JoinDistribution.getSupportedJoinTypes().contains(joinType)) {
+
+        final String SALT_UDF = "salt";
```

Comment:
capitalized variables should be static (I thought checkstyle would complain about this?)

Reply:
It did...I fixed it in the following commit